spec/lib/dispatcher_spec.rb

```diff
@@ -1,9 +1,8 @@
 describe Dispatcher do
   let(:proposal) { create(:proposal) }
 
-  describe '.deliver_new_proposal_emails' do
-    it "uses the LinearDispatcher for linear approvals" do
-      proposal.flow = 'linear'
+  describe ".deliver_new_proposal_emails" do
+    it "uses the LinearDispatcher for non-NCR approvals" do
```

Comment:
I am not sure. One of the challenges of having the concept of parallel steps without using them is that we don't have test cases on the integration level for the situation where we are using them.